hypothesis.py

```diff
@@ -0,0 +1,132 @@
+from dataclasses import dataclass
+import raster
+import scipy.stats
+import math
+import pandas as pd
+
+_LONGITUDE_COLUMN_NAME = 'long'
+_LATITUDE_COLUMN_NAME = 'lat'
+_FRAUDULENT_COLUMN_NAME = 'fraud'
+
+
+@dataclass
+class HypothesisTest:
+    '''
+    Represents a hypothesis test of a sample to an isoscape
+    '''
+    longitude: float
+    latitude: float
+    p_value: float
+    p_value_threshold: float
+
+
+def sample_ttest(longitude: float,
+                 latitude: float,
+                 isotope_values: pd.Series,
+                 means_isoscape: raster.AmazonGeoTiff,
+                 variances_isoscape: raster.AmazonGeoTiff,
+                 sample_size_per_location: int,
+                 p_value_target: float):
+    '''
+    longitude: Of the sample
+    latitude: Of the sample
+    isotope_values: Of the sample
+    means_isoscape: Isoscape that maps geographic coordinates to a mean isotope value.
+    variances_isoscape: Isoscape that maps geographic coordinates to the variance of
+                        isotope valuesat that location.
+    sample_size_per_location: Number of samples per geographic location used to calculate
+                              mean and variance in isoscapes.
+    p_value_target: desired p_value for the t-test (e.sample_data: 0.05)
+    '''
+    if isotope_values.size <= 1:
+        raise ValueError  # Isotope values needs to be more than 1.
+
+    isotope_mean = isotope_values.mean()
+    isotope_variance = isotope_values.var()*(isotope_values.size /
+                                             (isotope_values.size - 1))
+    isotope_sample_count = isotope_values.size
+
+    # Values from prediction.
+    predicted_isotope_mean = raster.get_data_at_coords(
+        means_isoscape, longitude, latitude, 0)
+    predicted_isotope_variance = raster.get_data_at_coords(
+        variances_isoscape, longitude, latitude, 0)
+    predicted_isotope_sample_count = sample_size_per_location
+
+    # t-student Test
+    _, p_value = scipy.stats.ttest_ind_from_stats(
```

Comment:
Can we use the `formal-parameter-name = argument-value` syntax here for clarity?